doc/sphinx/source/gpu.md

```diff
@@ -0,0 +1,22 @@
+# GPU Development
+
+Runtime selection of libCEED backends allows users to use CPU backends for easier debugging.
+Code that produces correct results with CPU backends will produce correct results on GPU backends, provided that JiT and memory access assumptions of the libCEED API are respected.
+
+## JiT Compilation
+
+The filepath to the user source code is passed in {c:func}`CeedQFunctionCreateInterior` as the `source` argument.
+This filepath should typically be an absolute path to ensure the JiT compilation can locate the source file.
```

Comment:
Maybe also mention that the `CEED_QFUNCTION` automatically creates this path in a string variable?

Reply:
added